pkg/api/dashboard.go

```diff
@@ -306,6 +308,26 @@ func (hs *HTTPServer) PostDashboard(c *models.ReqContext) response.Response {
 	if err := web.Bind(c.Req, &cmd); err != nil {
 		return response.Error(http.StatusBadRequest, "bad request data", err)
 	}
+
```

Comment:
Should the feature flag check be here?  with no flag set, i don't think we want to touch any validation path at all

Reply:
Kinda implied by other comments, but replying here to to not leave a reasonable question bare: writing the code in this way is about separating the responsibility of where the validation check should be executed (here) with exactly what the validation check _is_ (shared by schema, which defines the nature of validation, and feature flag, which decides whether to make it a no-op)